src/lib/button/button.scss

```diff
@@ -51,7 +51,7 @@
 }
 
 // The text and icon should be vertical aligned inside a button
-.mat-button, .mat-raised-button, .mat-icon-button {
+.mat-button, .mat-raised-button, .mat-icon-button, .mat-fab, .mat-mini-fab {
```

Comment:
Does this change the `color` in any circumstances?

Reply:
`mat-icon`